apps/dashboard/app/views/widgets/pinned_apps/_app.html.erb

```diff
@@ -1,26 +1,30 @@
 <%- link = app.links.first -%>
 <%- link_data = {:method => "get"}.merge link.data -%>
+<%- tile_data = link.tile -%>
+
 <div class="col-sm-3 col-md-3 app-launcher-container">
   <div class="app-launcher app-launcher-hover" data-toggle="launcher-button">
   <%=
     link_to(
       link.url.to_s,
-      class: "",
+      class: tile_data[:css_class],
```

Comment:
Can we change this back to `border_color`? I think that's going to resonate more with customers.  Or find some tricky way to alias `css_class` to `border_color`.  

```suggestion
      class: tile_data[:border_color],
```

I mean strictly speaking, it is css classes for a link, but realistically we only want folks changing border colors.

Reply:
OK. Will update.`border_color` is more to the point, advance user/admins will understand that is a CSS class.